packages/react-native-navigation/src/rum/instrumentation/DdRumReactNativeNavigationTracking.tsx

```diff
@@ -8,19 +8,26 @@ import React from 'react'
 import { ComponentDidAppearEvent, Navigation } from 'react-native-navigation';
 import { DdRum } from '@datadog/mobile-react-native';
 
+
+
+
+export type ViewNamePredicate = (trackedView: any, trackedName: string) => string | null
```

Comment:
we need to document this public type, because I think `trackedView` needs an explanation, like it is not exactly the view, but some view-related context, which is in case of `react-native-navigation` is component event, but not component itself.